### DISCUSSION TITLE
Grabaciones 1era Sesión Modulo 1

### DISCUSSION THREAD

**Discussion by MGARCIA58** [Launch-X-Latam/MisionFrontEnd#94] [Q&A] [answered]
Hola!

¿En donde se puede ver la grabación de la sesión del día miercoles 28 de septiembre?

En MS Teams, en el canal de general, en la pestaña de grabaciones solo se encuentran los 3 primeros vídeos y en el canal de módulo 1 no veo alguna pestaña de grabaciones...

**Comment by iam-leo** [comment, marked as answer]
En esos 3 videos que mencionas hay uno que dice "Live de Introducción a FrontEnd" ese es el correspondiente al miércoles 28. Lo puedes ver desde la pestaña de grabaciones o entrar en el canal de modulo 1 y buscar en publicaciones el post que contenga el mismo titulo, ahí vendrá el link.

**Reply by MGARCIA58** [reply]
Estaba bastante seguro que ese no era, pero tienes razón, muchas gracias...!

**Reply by iam-leo** [reply]
De nada compa! Otra forma que acabo de encontrar, es entrar en la pestaña de grabaciones y le das click en el icono rojo que parece una flecha  
![screen_04](https://user-images.githubusercontent.com/79548542/193340179-4e03d7c4-2f43-42b2-bd78-c927a7de6dad.PNG)
eso te llevará al canal de Misión Frontend en microsoft streams y ahí te saldrán ya los videos con las fechas en la que se transmitieron.
